src/Sulu/Bundle/CategoryBundle/Admin/CategoryContentNavigationProvider.php

```diff
@@ -24,7 +24,7 @@ public function getNavigationItems(array $options = [])
         $details = new ContentNavigationItem('content-navigation.category.details');
         $details->setAction('details');
         $details->setComponent('categories@sulucategory');
-        $details->setComponentOptions(['display' => 'form']);
+        $details->setComponentOptions(['display' => 'details']);
```

Comment:
Didn't we want to get rid of this option? And just start the right component immediately?